airbyte-integrations/connectors/source-fastbill/source_fastbill/source.py

```diff
@@ -0,0 +1,320 @@
+#
+# Copyright (c) 2022 Airbyte, Inc., all rights reserved.
+#
+
+
+from abc import ABC
+from typing import Any, Iterable, List, Mapping, MutableMapping, Optional, Tuple, Union
+
+import requests
+from airbyte_cdk.models import SyncMode
+from airbyte_cdk.sources import AbstractSource
+from airbyte_cdk.sources.streams import Stream
+from airbyte_cdk.sources.streams.http import HttpStream
+from airbyte_cdk.sources.streams.http.auth import BasicHttpAuthenticator
+from requests.auth import HTTPBasicAuth
+class FastbillStream(HttpStream, ABC):
+
+    def __init__(self, *args, username: str = None, api_key: str = None,**kwargs):
+        super().__init__(*args, **kwargs)
+        self._username = username
+        self._api_key = api_key
+
+    API_OFFSET_LIMIT = 100
+
+    @property
+    def http_method(self) -> str:
+        return "POST"
+    url_base = " https://my.fastbill.com/api/1.0/api.php"
```

Comment:
Please check the code formatting.

Reply:
Moved it to the top of the class instead